src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,446 @@
+which is REQUIRED for all MR data if PET data is also present in the dataset
+(see also [PET-MRI correspondence](../99-appendices/13-cross-modality-correspondence.md#pet-mri-correspondence)).
+In the case of studies using combined PET/fMRI,
+subject-specific tasks may be carried out during the acquisition within the same session.
+If the same task is recorded with both modalities,
+the same `task-<label>` entity SHOULD be used.
+For further details, see
+[Task (including resting state) imaging data](./01-magnetic-resonance-imaging-data.md#task-including-resting-state-imaging-data).
+
+In addition to the imaging data (`*.nii`) a `_pet.json` sidecar file needs to be provided.
+The included metadata are divided into sections described below.
+
+### PET sidecar JSON (`*_pet.json`)
+
+#### Information
+
+This section is mandatory and contains general information about the imaging experiment.
+
+Some of the fields are marked optional (MAY), for example anaesthesia; for those fields
+the [BIDS validator](https://github.com/bids-standard/bids-validator) will not throw an error even if they are not present.
+Note, although bodyweight is a recommended information in (Knudsen et al. 2020, [doi:10.1177/0271678X20905433](https://doi.org/10.1177/0271678X20905433)),
+this consists of meta information at the participant level and should hence
+be part of the `participants.tsv` or `session.tsv` file in case of multiple measurements.
+
+| **Key name**              | **Requirement level** | **Data type**                          | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| ------------------------- | --------------------- | -------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Manufacturer              | REQUIRED              | [string][]                             | Scanner manufacturer (for example, "Siemens").                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
Tiny formatting detail

checked in other pages of the spec and it seems that in the key description tables, most "string" value examples are formatted as follow:

```
Scanner manufacturer (for example, `Siemens`).
```

and not like 

```
Scanner manufacturer (for example, "Siemens").
```

This is more of a soft rule. 

I can push a commit or open a PR for that if it helps.

Reply:
Actually had a quick chat with the BIDS maintainers.

We sort of agree that there some good about the way the examples are written in this BEP and so the best way to write examples in tables would be the following:

```
Scanner manufacturer (for example, `"Siemens"`).
```

As this is currently not the case in pretty much the rest of the spec, I will open an issue to keep track of this.